pkg/broker/broker.go

```diff
@@ -76,23 +82,42 @@ func (br *Broker) ListenForClients() {
 		case sb := <-br.NewClients:
 			br.clients[sb] = true
 			cc, ok := br.clientsCount.Load(sb.ID)
-			if ok {
+			if ok && cc.(int) > 0 {
 				br.clientsCount.Store(sb.ID, cc.(int)+1)
 			} else {
 				br.clientsCount.Store(sb.ID, 1)
+				br.CloseClientC[sb.ID] = make(chan struct{})
+				tl, err := tailer.New(sb.Extra)
+				go func() {
+					br.Log.Printf("[tailer] Starting for %s", sb.ID)
+
+					scanner := bufio.NewScanner(tl)
```

Comment:
Since this variable is only used in the subsequent line, it could be renamed to the shorter "s".

Reply:
fixed